opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/netsnmp-graph.properties

```diff
@@ -310,6 +391,67 @@ report.netsnmp.cpuUsageFull.command=--title="CPU Usage" \
  GPRINT:cpuinterrupt:MIN:"Min  \\: %8.2lf %s" \
  GPRINT:cpuinterrupt:MAX:"Max  \\: %8.2lf %s\\n"
 
+report.netsnmp.cpuUsageFuller.name=CPU Usage
```

Comment:
The report `netsnmp.cpuUsageFuller` is missing on top of the file in the report section.

Reply:
fixed